features/text_components/publication_status.feature

```diff
@@ -0,0 +1,27 @@
+@311
+Feature: Only published text components should used to generate a report
+  As a reporter
+  I want to have a flag for text.components (draft, fact-checked, published) and only published text components should be included in the report
+  In order to add new text components without affecting the result of the live report
```

Comment:
Try to explain the value: Why does sb. want to add a text component without affecting the report? Maybe to show it to other team members?